### DISCUSSION TITLE
Welcome to CypressTest Discussions!

### DISCUSSION THREAD

**Discussion by adityamali12** [adityamali12/CypressTest#8] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by adityamali12** [comment]
Test

**Reply by AdityaNow** [reply, marked as answer]
test
